### PR TITLE
Edit labels

### PR DESCRIPTION
**Pull request by raykyri**
Copy edits to chain event labels.

If there are no issues this can be merged. We should bump the version and push to npm too.

### CODE REVIEW

File: src/substrate/filters/labeler.ts [hunk 1]
```diff
@@ -90,21 +90,21 @@ export const Label: LabelerFilter = (
       const { authorityId } = data;
       return {
         heading: 'Heartbeat Received',
-        label: ` A new heartbeat was received from ${fmtAddr(authorityId)}.`,
+        label: `A new heartbeat was received from ${fmtAddr(authorityId)}.`,
```
Comment (jnaviask) [inline src/substrate/filters/labeler.ts:93]: I would just say "A heartbeat" rather than "A new heartbeat".

File: src/substrate/filters/labeler.ts [hunk 2]
```diff
@@ -90,21 +90,21 @@ export const Label: LabelerFilter = (
...
       };
     }
     case EventKind.SomeOffline: {
       const { sessionIndex } = data;
       return {
         heading: 'Some Offline',
-        label: `At the end of the session: ${sessionIndex}, at least one validator was found to be offline.`,
+        label: `At the end of session ${sessionIndex}, at least one validator was found to be offline.`,
```
Comment (jnaviask) [inline src/substrate/filters/labeler.ts:100]: Maybe reword to "At least one validator was found to be offline at the end of session X". But we may want to avoid passive voice here and just say "At least one validator went offline during session X".

File: src/substrate/filters/labeler.ts [hunk 3]
```diff
@@ -90,21 +90,21 @@ export const Label: LabelerFilter = (
...
       };
     }
     case EventKind.AllGood: {
       const { sessionIndex } = data;
       return {
         heading: 'All Good',
-        label: `At the end of the session: ${sessionIndex}, no offence was committed.`,
+        label: `At the end of session ${sessionIndex}, no offence was committed.`,
```
Comment (jnaviask) [inline src/substrate/filters/labeler.ts:107]: Maybe reword to "No offences were committed at the end of session X". Again, though, we may want to avoid passive voice and say "No validators committed offences during session X".

File: src/substrate/filters/labeler.ts [hunk 4]
```diff
@@ -159,7 +159,7 @@ export const Label: LabelerFilter = (
       const { stash, controller, amount } = data;
       return {
         heading: 'Bonded',
-        label: `You bonded ${balanceFormatter(amount)} from controller ${fmtAddr(controller)} to stash ${fmtAddr(stash)}.`,
+        label: `${balanceFormatter(amount)} was bonded by controller ${fmtAddr(controller)} on stash ${fmtAddr(stash)}.`,
```
Comment (jnaviask) [inline src/substrate/filters/labeler.ts:162]: The reason these are "you" is that they're only supposed to be sent to the single address specified in the CWEvent's `includeAddresses`. But if we want these public, then yeah this change makes sense.

**Review by jnaviask — COMMENTED** [review]
Left a few comments with ideas but seems fine.

File: src/substrate/filters/labeler.ts (hunk 4 above)
Comment (raykyri) [reply src/substrate/filters/labeler.ts:162]: Indeed we're using them publicly right now. I think we can figure out a way to inject "Notification about your address:" before the label that will let the label be used in both cases.
